index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+
+
+<body>
+
+    <!-- header -->
+
+    <div class="container"> <!-- wrapper -->
+
+        <header>
+            <nav class="navbar">
+                <div class="logo">My Logo</div>
+
+                <div class="bar-toggle">
+                    <i class="fa fa-bars"></i>
+                </div>
+
+                <!-- navlinks -->
+            </nav>
+        </header>
+
+        <!-- small-container -->
+        <main>
+            <section class="small-container">
+                <h1>
+                    I’m Ayaz <br>
+                    Glad to see<br> you!
+                </h1>
+                <p class="paragraph-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.
+                </p>
+
+                <a href="#" class="social-link">
+                    Let's connect
+                </a>
+
+                <!-- social-media-link -->
+
+                <ul class="social-media-link">
+                    <li class="social-icon">
+                        <a href="#">
+                            <img src="./images/Vector-1.png" alt="Image missing">
+                        </a>
+                    </li>
+
+                    <li class="social-icon">
+                        <a href="#">
+                            <img src="./images/Linkedin icon.png" alt="Image missing">
+                        </a>
+                    </li>
+
+                    <li class="social-icon">
+                        <a href="#">
+                            <img src="./images/vector 4.png" alt="Image missing">
+                        </a>
+                    </li>
+
+                    <li class="social-icon">
+                        <a href="#">
+                            <img src="./images/Vector.png" alt="Image missing">
+                        </a>
+                    </li>
```

Comment:
- [ ] According to best practises, kindly use descriptive names for all your `alt` attributes. 👍 The `alt` attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the `src` attribute, or if the user uses a screen reader).